docs/API.md

```diff
@@ -493,4 +494,18 @@ Parameters:
   * uid : user id
   * type : type of reminder
 
+## .createEmailBounce(body) ##
+
+Record when an email bounce has occurred.
+
+Parameters:
+
+* body: (object)
+  * email: A string of the email address that bounced
+  * bounceType: The bounce type ([`'Permanent'` or `'Transient'`])
+  * bounceSubType: The bounce sub type string
+  * template: The string name of the template that was used for the sent email
+  * lang: The language of the sent email
```

Comment:
I assumed they'd be mostly just useful for metrics purposes. If we're fine with the logging that already occurs, we can pull this out. I don't expect that it would really affect our sending of future email in the auth-server (as in, I don't expect us to check "oh, it failed 3 times with French, suggest English!" or something weird like that).